server/bin/www

```diff
@@ -31,34 +32,60 @@ const server = http.createServer(app);
 
 const io = require("socket.io")(server);
 
+io.use((socket, next) => {
```

Comment:
Small nit: this appear to be for another ticket 🤔 In the future, let's make sure only relevant PR showed up in the review.

Reply:
Thank you for catching that! I'll make sure it doesn't happen again.